src/e2e-test/resources/errorMessage.properties

```diff
@@ -13,3 +13,7 @@ errorMessagePubSubMaxBatchSizeField=Invalid maximum batch size. Ensure the value
 errorMessagePubSubDelayThresholdField=Invalid delay threshold for publishing a batch. Ensure the value is a positive number.
 errorMessagePubSubRetryTimeout=Invalid max retry timeout for retrying failed publish. Ensure the value is a positive number.
 errorMessagePubSubErrorThreshold=Invalid error threshold for publishing. Ensure the value is a positive number.
+errorMessageIncorrectBQChunkSize=Value must be a multiple of 262144.
+errorMessageIncorrectBQSinkDataset=Dataset name can only contain letters (lower or uppercase), numbers and '_'.
```

Comment:
Create common error message -
errorMessageIncorrectBQProperty=PROPERTY name can only contain letters (lower or uppercase), numbers and '_'.